fee/src/lib.rs

```diff
@@ -17,21 +15,15 @@ use {
 #[derive(Copy, Clone)]
 #[cfg_attr(
     feature = "svm-internal",
-    field_qualifiers(
-        remove_rounding_in_fee_calculation(pub),
-        enable_secp256r1_precompile(pub)
-    )
+    field_qualifiers(enable_secp256r1_precompile(pub))
```

Comment:
@tao-stones I have asked Pankaj to revert the change that introduced this change (only became aware of it due to this PR).
Would prefer a clean revert commit, so going to wait for his response to my comment before approving this PR.

Reply:
Here's the revert: https://github.com/anza-xyz/agave/pull/4694

Reply:
rebased, thanks @pgarg66